db/compaction/compaction.cc

```diff
@@ -566,6 +567,52 @@ std::unique_ptr<CompactionFilter> Compaction::CreateCompactionFilter(
       context);
 }
 
+std::pair<std::vector<Slice>, std::vector<uint64_t>>
+Compaction::CreateSegmentsForLevel(int in_level) const {
```

Comment:
why call it `in_level`?

Reply:
This creates the segment of the input level, so the argument is named `in_level`, perhaps a plain `level` might be fine. 🤔️